src/APIFunctions/Mailer.js

```diff
@@ -85,3 +85,24 @@ export async function addEventToCalendar(newEvent, token) {
     });
   return status;
 }
+
+/**
+ * Verify a user's email after the visiting the link sent to their email.
+ * @param {string} email - The user's email
+ * @param {string} hashedId - A hashed value of the user's mongoDB _id field
+ * @returns {ApiResponse} Containing any error information related to the
+ * request
+ */
+export async function validateVerificationEmail(email, hashedId) {
```

Comment:
this function is already in [`src/APIFunctions/Auth.js:110`](https://github.com/SCE-Development/Core-v4/blob/f82138a1e5bf97c3110472782abf953e7c202bd0/src/APIFunctions/Auth.js#L110), you can revert the changes in this file